### PR TITLE
Simplify prefect-aws and prefect-dbt docs index pages

### PR DESCRIPTION
**Pull request by discdiver**
Simplifies and clarifies the intro page for prefect-aws.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] If this pull request adds new functionality, it includes unit tests that cover the changes
- [X] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed.

For new functions or classes in the Python SDK:

- [ ] This pull request includes helpful docstrings.
- [ ] If a new Python file was added, this pull request contains a stub page in the Python SDK docs and an entry in `mkdocs.yml` navigation.

### CODE REVIEW

File: docs/integrations/prefect-aws/index.md [hunk 1]
```diff
@@ -53,26 +54,34 @@ AwsCredentials(
 ).save("BLOCK-NAME-PLACEHOLDER")
 ```
 
-Congrats! You can now load the saved block to use your credentials in your Python code:
+Note that under the hood, prefect is using the Boto3 library.
```
Comment (billpalombi) [inline docs/integrations/prefect-aws/index.md:57]:
```suggestion
Note that, under the hood, prefect is using the Boto3 library.
```

File: docs/integrations/prefect-aws/index.md [hunk 2]
```diff
@@ -113,23 +123,20 @@ def secrets_manager_flow():
...
 
-For more tips on how to use blocks and tasks in Prefect integration libraries, check out the [docs](https://docs.prefect.io/integrations/usage/)!
+See the [ECS guide](/ecs_guide/) for a walkthrough of using ECS in a hybrid work pool.
 
-For more information about how to use Prefect, please refer to the [Prefect documentation](https://docs.prefect.io/).
+If you're using Prefect Cloud and your organization's security posture allows storing credentials in blocks, [ECS push work pools](https://docs.prefect.io/latest/guides/deployment/push-work-pools/#__tabbed_1_1) are a great option.
+They provide all the benefits of ECS with a quick setup and no worker needed.
 
-If you encounter any bugs while using `prefect-aws`, feel free to open an issue in the [`prefect`](https://github.com/PrefectHQ/prefect) repository.
+## Next steps
```
Comment (billpalombi) [inline docs/integrations/prefect-aws/index.md:140]: I don't think we need this

File: docs/integrations/prefect-aws/index.md (hunk 1 above)
Comment (discdiver) [reply docs/integrations/prefect-aws/index.md]: TY, reworded and pushed

**Review by billpalombi — APPROVED** [review]
lgtm

File: docs/integrations/prefect-aws/index.md (hunk 2 above)
Comment (discdiver) [reply docs/integrations/prefect-aws/index.md]: Changed to resources and moved a bit more, hoping to follow a similar format for other libraries, so trying to keep it flexible. PR now includes a few updates to prefect-dbt to try to align with these changes in prefect-aws.

File: docs/integrations/prefect-aws/index.md [hunk 3]
```diff
@@ -53,26 +54,34 @@ AwsCredentials(
...
 ```
 
-Congrats! You can now load the saved block to use your credentials in your Python code:
+Prefect is using the Boto3 library under the hood.
+To find credentials for authentication, any data not provided to the block is sourced at runtime in the order shown in the [Boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials).
+Prefect creates the session object using the values in the block and then, any missing values follow the sequence in the Boto3 docs.
+
+See an example of using the `AwsCredentials` block with [AWS Secrets Manager](#aws-secrets-manager) with third-party services without storing credentials in the block itself in [this guide](/guides/secrets/).
+
+Here's how to load the saved credentials in a Python script:
```
Comment (billpalombi) [inline docs/integrations/prefect-aws/index.md:63]:
```suggestion
Here's how to load the saved credentials:
```

**Review by billpalombi — APPROVED** [review]
One nit, otherwise lgtm

**Review by billpalombi — APPROVED** [review]
third times the charrm

File: docs/integrations/prefect-aws/index.md [hunk 4]
```diff
@@ -53,26 +54,34 @@ AwsCredentials(
 ).save("BLOCK-NAME-PLACEHOLDER")
 ```
 
-Congrats! You can now load the saved block to use your credentials in your Python code:
+Prefect is using the Boto3 library under the hood.
+To find credentials for authentication, any data not provided to the block is sourced at runtime in the order shown in the [Boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials).
```
Comment (discdiver) [inline docs/integrations/prefect-aws/index.md:58]:
```suggestion
To find credentials for authentication, any data not provided to the block are sourced at runtime in the order shown in the [Boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials).
```

File: docs/integrations/prefect-dbt/index.md [hunk 5]
```diff
@@ -268,10 +231,56 @@ dbt_core_operation = DbtCoreOperation(
...
+
+### Additional installation options
+
+Additional installation options for dbt Core with BigQuery, Snowflake, and Postgres are shown below.
+
+To install with additional functionality for dbt Core and Snowflake profiles:
+
+<div class = "terminal">
+```bash
+pip install "prefect-dbt[snowflake]"
```
Comment (discdiver) [inline docs/integrations/prefect-dbt/index.md:256]:
```suggestion
pip install -U "prefect-dbt[snowflake]"
```

File: docs/integrations/prefect-dbt/index.md [hunk 6]
```diff
@@ -268,10 +231,56 @@ dbt_core_operation = DbtCoreOperation(
...
+```bash
+pip install "prefect-dbt[bigquery]"
+```
+</div>
+
+To install with additional functionality for dbt Core and Postgres profiles:
+
+<div class = "terminal">
+```bash
+pip install "prefect-dbt[postgres]"
```
Comment (discdiver) [inline docs/integrations/prefect-dbt/index.md:272]:
```suggestion
pip install -U "prefect-dbt[postgres]"
```

File: docs/integrations/prefect-dbt/index.md [hunk 7]
```diff
@@ -268,10 +231,56 @@ dbt_core_operation = DbtCoreOperation(
...
+```bash
+pip install -U "prefect-dbt[snowflake]"
+```
+</div>
+
+To install with additional functionality for dbt Core and BigQuery profiles:
+
+<div class = "terminal">
+```bash
+pip install "prefect-dbt[bigquery]"
```
Comment (discdiver) [inline docs/integrations/prefect-dbt/index.md:264]:
```suggestion
pip install -U "prefect-dbt[bigquery]"
```

File: docs/integrations/prefect-dbt/index.md [hunk 8]
```diff
@@ -1,22 +1,54 @@
...
+```bash
+pip install -U prefect-dbt
+```
+</div>
+
+Install with additional functionality for dbt Core (CLI):
+
+<div class = "terminal">
+```bash
+pip install -U "prefect-dbt[cli]"
```
Comment (seanpwlms) [inline docs/integrations/prefect-dbt/index.md:27]: I think this section can be omitted.  `dbt_core` is not an optional dependency, currently.

**Pull request merged**